Source/WebCore/html/HTMLAttachmentElement.cpp

```diff
@@ -469,13 +469,13 @@ void HTMLAttachmentElement::attributeChanged(const QualifiedName& name, const At
 
     if (name == actionAttr) {
         if (m_actionTextElement)
-            m_actionTextElement->setTextContent(String(newValue.string()));
+            m_actionTextElement->setTextContent(String(attachmentActionForDisplay()));
```

Comment:
Nit: I guess this is just a matter of style, but I think it's more common to see `someAtom.string()` rather than `String(someAtom)` in the codebase.

Reply:
> however setTextContent takes an rvalue-ref

OK, I didn't notice that.